plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -1700,10 +1799,9 @@ function VocabularyBuilderWidget:onSwipe(arg, ges_ev)
 end
 
 function VocabularyBuilderWidget:onMultiSwipe(arg, ges_ev)
-    -- For consistency with other fullscreen widgets where swipe south can't be
-    -- used to close and where we then allow any multiswipe to close, allow any
-    -- multiswipe to close this widget too.
-    self:onClose()
+    -- since swipe south is closing, we reload items here
+    self:reloadItems()
```

Comment:
If I were a Vocabulary Builder user, I would be using multiswipe to quit it, and I would really complain loudly about the inconsistency :)
As I am not, I will just complain silently :)
(If it were me, and I dunno how useful reloading is, I would probably try to match all 4 of "north/west/south/east" in the gesture name, expecting the user to draw a square/circle in any direction/order, like a reload icon:
![image](https://user-images.githubusercontent.com/24273478/205994860-677a3313-44a2-4873-b6e4-242f9dc7c5e5.png) )

Reply:
Good idea. Your silent complaint has been heard :) and the circle gesture done (though not sure if it is the preferred way to match directions)

Reply:
Thanks :)
My "restart koreader" gesture is south/west/north/east - so you could allow starting the circle with south. (Starting it with north is quite a bit less natural and harder for my brain to do :)

Reply:
Done.